src/CbcSolverInterface.jl

```diff
@@ -2,8 +2,25 @@ module CbcMathProgSolverInterface
 
 using Cbc.CbcCInterface
 
-importall MathProgBase.SolverInterface
+import MathProgBase
+const MPB = MathProgBase.SolverInterface
 
+import MathProgBase.SolverInterface: getnodecount,
+    getobjbound,
+    getobjgap,
+    getobjval,
+    getrawsolver,
+    getsense,
+    getsolution,
+    getvartype,
+    loadproblem!,
+    numconstr,
+    numvar,
+    optimize!,
+    setsense!,
+    setvartype!,
+    status,
+    writeproblem
 
```

Comment:
These imports are only needed for the corresponding exports, but maybe those should go away? Seems to have done so in Clp at least.

Reply:
Yes, please remove the re-exports and this list of imports.

Reply:
Done.